src/gui/editors/SongEditor.cpp

```diff
@@ -293,6 +302,98 @@ void SongEditor::scrolled( int new_pos )
 
 
 
+void SongEditor::selectRegionFromPixels(int xStart, int xEnd)
+{
+	if (!m_selectRegion)
+	{
+		m_selectRegion = true;
+
+		//deselect all tcos
+		for (auto &it : findChildren<selectableObject *>()) { it->setSelected(false); }
+
+		rubberBand()->setEnabled(true);
+		rubberBand()->show();
+
+		//we save the position of scrollbars, mouse position and zooming level
+		m_origin = QPoint(xStart, 0);
+		m_scrollPos = QPoint(m_leftRightScroll->value(), contentWidget()->verticalScrollBar()->value());
+		m_currentZoomingValue = zoomingModel()->value();
+
+		//calculate the song position where the mouse was clicked
+		m_rubberbandStartMidipos = MidiTime((xStart - m_trackHeadWidth)
+											/ pixelsPerTact() * MidiTime::ticksPerTact())
+											+ m_currentPosition;
```

Comment:
You should set `m_rubberBandStartTrackview` to 0 as well. If I select some TCOs in track views and then from the timeline, some TCOs are not selected although the rubberband displays correctly.

Reply:
fixed it.